--- a/Makefile
+++ b/Makefile
@@ -19,21 +19,19 @@
 #
 # $Id: Makefile 312 2008-06-15 18:09:42Z Joerg Mayer $
 
-DESTDIR=
-PREFIX=/usr/local
-ETCDIR=/etc/vpnc
-BINDIR=$(PREFIX)/bin
-SBINDIR=$(PREFIX)/sbin
-MANDIR=$(PREFIX)/share/man
-DOCDIR=$(PREFIX)/share/doc/vpnc
+DESTDIR =
+PREFIX ?= /usr/local
+ETCDIR  = /etc/vpnc
+BINDIR  = $(PREFIX)/bin
+SBINDIR = $(PREFIX)/sbin
+MANDIR  = $(PREFIX)/share/man
+DOCDIR  = $(PREFIX)/share/doc/vpnc
 
-SRCS = sysdep.c vpnc-debug.c isakmp-pkt.c tunip.c config.c dh.c math_group.c supp.c decrypt-utils.c
-BINS = vpnc cisco-decrypt
-OBJS = $(addsuffix .o,$(basename $(SRCS)))
+SRCS    = sysdep.c vpnc-debug.c isakmp-pkt.c tunip.c config.c dh.c math_group.c supp.c decrypt-utils.c
+BINS    = vpnc cisco-decrypt
+OBJS    = $(addsuffix .o,$(basename $(SRCS)))
 BINOBJS = $(addsuffix .o,$(BINS))
 BINSRCS = $(addsuffix .c,$(BINS))
-VERSION := $(shell sh mk-version)
-RELEASE_VERSION := $(shell cat VERSION)
 
 # The license of vpnc (Gpl >= 2) is quite likely incompatible with the
 # openssl license. Openssl is currently used to provide certificate
@@ -50,11 +48,11 @@
 #OPENSSL_GPL_VIOLATION = -DOPENSSL_GPL_VIOLATION
 #OPENSSLLIBS = -lcrypto
 
-CC=gcc
+CC ?= gcc
 CFLAGS ?= -O3 -g
 CFLAGS += -W -Wall -Wmissing-declarations -Wwrite-strings
-CFLAGS +=  $(shell libgcrypt-config --cflags)
-CPPFLAGS += -DVERSION=\"$(VERSION)\" $(OPENSSL_GPL_VIOLATION)
+CFLAGS += $(shell libgcrypt-config --cflags)
+CPPFLAGS += $(OPENSSL_GPL_VIOLATION)
 LDFLAGS ?= -g
 LDFLAGS += $(shell libgcrypt-config --libs) $(OPENSSLLIBS)
 
