--- a/acinclude.m4
+++ b/acinclude.m4
@@ -232,15 +232,15 @@
 
 if test "$TARGET" = "cdk"; then
 	datadir="\${prefix}/share"
-	sysconfdir="\${prefix}/etc"
-	localstatedir="\${prefix}/var"
+	sysconfdir="/etc"
+	localstatedir="/var"
 	libdir="\${prefix}/lib"
-	mntdir="\${prefix}/mnt"
+	mntdir="/mnt"
 	targetdatadir="\${TARGET_PREFIX}/share"
-	targetsysconfdir="\${TARGET_PREFIX}/etc"
-	targetlocalstatedir="\${TARGET_PREFIX}/var"
+	targetsysconfdir="/etc"
+	targetlocalstatedir="/var"
 	targetlibdir="\${TARGET_PREFIX}/lib"
-	targetmntdir="\${TARGET_PREFIX}/mnt"
+	targetmntdir="/mnt"
 else
 	mntdir="/mnt" # hack
 fi
